src/main/webapp/partials/instructorHelpSections.jsp

```diff
@@ -0,0 +1,64 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="sections">Sections</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="sectionsWhatIsItMeantFor">
+                    <h3>1. What are sections meant for?</h3>
+                </a>
+            </span>
+            <p>
+                Sections are meant to organize students where the course has significantly large number of students. It is mandatory for courses with more than 100 students to organize a student into a section.
+                You are allowed to have sections for courses with less than 100 students, but it is not mandatory.
+            </p>
+        </li>
+        <li>
+            <span class="text-bold">
+                <a name="sectionsEnrollingStudents">
+                    <h3>2. Enrolling students into sections</h3>
+                </a>
+            </span>
+            <p>
+                You can specify the section of each student at the time of enrollment. You just have to include a
+                <b>'Section'</b> column in the spreadsheet or the student data you mention in the text box.
+                <br>More information can be found on the course enrollment page.
+                Go to the
+                <b>'Courses'</b> page. You will see the list of courses, then click on
+                <b>'Enroll'</b> button in the last column of the row corresponding to the course to which you want to enroll students. Scroll down to the <b>'More Info'</b> section.
+            </p>
+        </li>
+        <li>
+            <span class="text-bold">
+                <a name="sectionsEditing">
+                    <h3>3. Editing a student's section</h3>
+                </a>
+            </span>
+            <p>
+                Go to the
+                <b>'Students'</b> page, click the panel heading for the relevant course. You will see a list of students for the course, then click
+                <b>'Edit'</b> button in the last column of the row corresponding to the student whose details you want to see.
```

Comment:
Follow the `Viewing student profiles` section but edit the last part:

`... corresponding to the student whose `~~details you want to see~~` section you want to edit.`